snuba/migrations/snuba_migrations/replays/0001_replays.py

```diff
@@ -0,0 +1,96 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Array,
+    Column,
+    DateTime,
+    IPv4,
+    IPv6,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+raw_columns: Sequence[Column[Modifiers]] = [
+    Column("replay_id", UUID()),
+    Column("sequence_id", UInt(16)),
+    Column("trace_ids", Array(UUID())),
```

Comment:
What are these trace_ids? Is it supposed to be a pointer to some other piece of data in one of our systems?

Reply:
https://github.com/getsentry/sentry-replay/issues/38#issuecomment-1110054329

replays can have N trace_ids, and each update may have N of them, and trace_id will be the link between them to start (we won't be doing any joins with them)

Reply:
on a per row basis it likely won't be more than 10